bin/kafka_bridge_run.sh

```diff
@@ -13,4 +13,9 @@ fi
 # Make sure that we use /dev/urandom
 JAVA_OPTS="${JAVA_OPTS} -Dvertx.cacheDirBase=/tmp/vertx-cache -Djava.security.egd=file:/dev/./urandom"
 
+# enabling OpenTelemetry with Jaeger by default
+if [ -n "$OTEL_SERVICE_NAME" ] && [ -z "$OTEL_TRACES_EXPORTER" ]; then
+  export OTEL_TRACES_EXPORTER="jaeger"
+fi
```

Comment:
aren't we doing this check in the `OpenTelemetryHandle` already (together with setting jaeger as default exporter)? Why here as well?

Reply:
Just in case you don't run Bridge via this script ... which you don't if you use IDE.

Reply:
Tbh the Strimzi bridge is supposed to run via the script. If you are not using it but the IDE, it's your responsibility to set the right env vars to make it working. I would remove the duplicated code and leave just the script imho.

Reply:
So removing the dup code in code, but leave the script?

Reply:
Yeah that was my suggestion